plugins/modules/ec2_instance_info.py

```diff
@@ -33,6 +33,11 @@
     required: false
     default: {}
     type: dict
+  uptime:
+    description:
+      - minimum running uptime in minutes of instances.  For example if uptime is 60, it would return all instances that have run more than 60 minutes.
```

Comment:
The name of the parameter does not communicate well it's behaviour. What about `minimum_uptime`?

Reply:
For brevity I like "uptime", but I agree that it's ambiguous.  One option would be to rename the option to minimum_uptime but add uptime as an alias.